src/index.md

```diff
@@ -7,7 +7,7 @@ title: "ByteZone Dev"
 
 ## Come hire us!
 
-Here at ByteZone we're a developement firm focused on building exceptional solutions so you can focus on your business.
+Here at ByteZone we're a development firm focused on building exceptional solutions so you can focus on your business.
```

Comment:
When rendered, is there now whitespace between this "element" and the next paragraph?

Reply:
No, but then again there isn't any now.

Reply:
Seems like we want that margin. Is this the PR to resolve that in?

Reply:
Yeah, let's add some default spacing. A layout PR seems like the appropriate place for that 👍

Reply:
Check it out now with the stack. That adds spacing between every set of tags in the main element (and anything else that adds that new class)